Altis_Life.Altis/core/functions/fn_numberText.sqf

```diff
@@ -1,28 +1,23 @@
 /*
     File: fn_numberText.sqf
-    Author: Karel Moricky
 
     Description:
-    Convert a number into string (avoiding scientific notation)
+    Convert a number into string (avoiding scientific notation) with comma formatting
 
     Parameter(s):
     _this: NUMBER
 
     Returns:
     STRING
 */
-private ["_number","_mod","_digots","_digitsCount","_modBase","_numberText"];
+params [["_number",0,[0]]]; 
+ 
+private _numberText = _number toFixed 0; 
+private _num = (count _numberText); 
 
-_number = [_this,0,0,[0]] call bis_fnc_param;
-_mod = [_this,1,3,[0]] call bis_fnc_param;
+for "_index" from (_num - 3) to 1 step -3 do { 
+     _numberText = (_numberText select [0, _index]) + "," + (_numberText select [_index]); 
```

Comment:
Could use format here. You may have used the plus operator here instead if it's faster, but I'm not sure whether thats why you used it here.